addons/web_editor/models/ir_qweb_fields.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class HTMLConverter(models.AbstractModel):
+    _inherit = 'ir.qweb.field.html'
+
+    @api.model
+    def attributes(self, record, field_name, options, values=None):
+        attrs = super().attributes(record, field_name, options, values)
+        if options.get('inherit_branding'):
+            field = record._fields[field_name]
+            if field.sanitize:
+                attrs['data-oe-sanitize'] = 1
+        return attrs
```

Comment:
I see that the original implementation of this method in base is checking `if not options['inherit_branding'] and not options['translate']:`, should not the translate option be considered here too?

Actually I am wondering if this really needs to be made as an inheritance, maybe we could just put that in base directly (maybe nothing should be in base TBH, I don't think the branding system is useful except for the web_editor so... but since it's in base, well, let's put everything in base?)

Could use @rdeodoo 's opinion on this whole PR :+1: (don't change anything before he comments back)